releases/geneva/compose-files/docker-compose-geneva.yml

```diff
@@ -0,0 +1,585 @@
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - vault-config:/vault/config
+    depends_on:
+      - logging
+
+  metadata:
+    image: edgexfoundry/docker-core-metadata-go:1.1.0
+    ports:
+      - "48081:48081"
+    container_name: edgex-core-metadata
+    hostname: edgex-core-metadata
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - vault-config:/vault/config
+    depends_on:
+      - logging
+
+  data:
+    image: edgexfoundry/docker-core-data-go:1.1.0
+    ports:
+      - "48080:48080"
+      - "5563:5563"
+    container_name: edgex-core-data
+    hostname: edgex-core-data
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - vault-config:/vault/config
+    depends_on:
+      - logging
+
+  command:
+    image: edgexfoundry/docker-core-command-go:1.1.0
+    ports:
+      - "48082:48082"
+    container_name: edgex-core-command
+    hostname: edgex-core-command
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - vault-config:/vault/config
+    depends_on:
+      - metadata
+
+  scheduler:
+    image: edgexfoundry/docker-support-scheduler-go:1.1.0
+    ports:
+      - "48085:48085"
+    container_name: edgex-support-scheduler
+    hostname: edgex-support-scheduler
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - vault-config:/vault/config
+    depends_on:
+      - metadata
+
+  app-service-rules:
+    image: edgexfoundry/docker-app-service-configurable:1.0.0
+    ports:
+      - "48100:48100"
+    container_name: edgex-app-service-configurable-rules
+    hostname: edgex-app-service-configurable-rules
+    networks:
+      edgex-network:
+        aliases:
+          - edgex-app-service-configurable-rules
+    environment:
+      <<: *common-variables
+      edgex_service: http://edgex-app-service-configurable-rules:48100
+      edgex_profile: rules-engine
+      Service_Host: edgex-app-service-configurable-rules
+      MessageBus_SubscribeHost_Host: edgex-core-data
+    depends_on:
+      - consul
+      - logging
+      - data
+
+  rulesengine:
+    image: edgexfoundry/docker-support-rulesengine:1.1.0
+    ports:
+      - "48075:48075"
+    container_name: edgex-support-rulesengine
+    hostname: edgex-support-rulesengine
+    networks:
+      edgex-network:
+        aliases:
+          - edgex-support-rulesengine
+    depends_on:
+      - app-service-rules
+
+  #  export-client:
```

Comment:
Can we remove this commented out section since `export-client` and `export-distro` have been removed and will not be an options for Geneva?

This applies to the other `docker-compose` files in this PR as well.

Reply:
Done

Reply:
This raises a concern -- are these files based on fuji release or on nightly-build?  They need to be based on nightly-build (and include all of the changes that may have been made since the fuji release branches were created).

Reply:
These files are based on Fuji and I left them there as place holders. I updated the [edgex-go, blackbox-testing, developer-scripts] PR tuple to use nightly-build.

Reply:
https://github.com/edgexfoundry/developer-scripts/pull/218#discussion_r377253535